### DISCUSSION TITLE
Broken repository in latest version

### DISCUSSION THREAD

**Discussion by tnbnicer** [community/community#23850] [New to GitHub]
<p>Hi.</p>
<p>I wonder if anyone has the authority to roll back a repo to the previous version.</p>
<p>In this case, the mod, the admin, or whoever owns (is that the right word?) the project, introduced changes that in the latest version/release break the software.</p>
<p>I’m not going to mention any names here. I don’t necessarily want anyone to get into trouble. Suffice to say that the project in question is fairly popular and has existed for many years.</p>
<p>It is defunct as of about 10 days ago, because a maintainer, perhaps unwittingly , added code that broke it. What can I do?</p>
<p>Note: I reported it as an issue (<span class="hashtag">#150</span>) yesterday in the main branch of the repo, however it goes beyond an issue.</p>
<p>Thanks.</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="tnbnicer" data-post="1" data-topic="203168">
<div class="title">
<div class="quote-controls"></div>
 tnbnicer:</div>
<blockquote>
<p>Note: I reported it as an issue (<span class="hashtag">#150</span>) yesterday in the main branch of the repo, however it goes beyond an issue.</p>
</blockquote>
</aside>
<p>If the code in some repository is broken, raising an issue there is what you should do. Fixing it (or not) is up to the people who own the repository.</p>
<p>What you can always do on your own is to use an older version instead of the latest commit, and possibly branch from there. :slightly_smiling_face:</p>

**Comment by tnbnicer** [comment, marked as answer]
<p>Thanks for replying with a smiley face. I’m trying to see the lighter side. A year and half ago I made some contributions to the project, when someone else was maintaining it. Importantly we discussed PRs first before implementing.</p>
<p>On Github the project has been there for a long long time (without giving too much away).</p>
<p>It is a shame to bear witness to the end this way. But I agree, older releases are available.  :slight_smile:</p>
